contents/blog/startup-ops-toolkit.md

```diff
@@ -1,102 +1,133 @@
 
-## Where to put the money
+In Europe, [Revolut](https://www.revolut.com/) is about the only digital bank that gets anywhere near Brex features-wise, although I find its platform a lot buggier.
 
-<iframe src="https://giphy.com/embed/3o6ZtpoNGDS4cZ8GWY" width="480" height="338" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/southparkgifs-3o6ZtpoNGDS4cZ8GWY">via GIPHY</a></p>
+> **Time suck alert:** I know what you're thinking – surely there's somewhere I can put my money to get some interest? The reality is that shopping around furiously to find some paltry 0.75% interest rate is highly unlikely to make any difference to your burn rate, and you're better off (unless you have $20m+) prioritizing having your cash easily accessible vs. trying to optimize for returns. I am not a financial advisor though - you should talk to one if you decide this is important.
 
-Put it in <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.brex.com/">Brex</a>, obviously (US only). They also make it incredibly easy to manage virtual credit cards, and their built-in expense management negates the need for something like Expensify. They don’t charge you anything.
+## 3. Keeping yourself organized
 
-In Europe, <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.revolut.com/">Revolut</a> are about the only digital bank that get anywhere near Brex features-wise, although I find their platform a lot more buggy.
+<iframe src="https://giphy.com/embed/xT1XGOGdyDrL2BTfxK" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
 
-I know what you're thinking - surely there's somewhere I can put my money to get some interest? The reality is that shopping around furiously to find some paltry 0.75% interest rate is highly unlikely to make any difference to your burn rate, and you're better off (in the early days) prioritizing having your cash easily accessible vs. trying to optimize for returns. I am not a financial advisor though - you should talk to one if you decide this is important.
+Now you've organized the two most important things (people, money), it's time to get the rest of your company running smoothly. Defining ops as ‘anything that is not building the product or talking to customers’, here is the list of other things you'll need to think about:
 
-## Ok, but what about…?
+- [Workable](https://www.workable.com/) for **recruitment**. $99 per role per month, pay as you go. Once you're hiring 3+ people a month, consider upgrading your account.
+ 
+- [HubSpot](https://www.hubspot.com/) for **sales and CRM**. Various tiers and combos available, starting from free. Free doesn’t mean you should start there, though. We like its Marketing Starter and Sales Pro products.
+ 
+- [Charlie](https://www.charliehr.com/) for **HR** things like recording salaries, time off, and contact details. $6 per employee per month. Charlie's UI looks janky, but it works.
```

Comment:
My understanding is we only use Charlie because we have non-US employees; Gusto can handle this all for us already